.github/workflows/main.yml

```diff
@@ -1,38 +1,72 @@
-name: OCCLUM CI
+name: Make Test
```

Comment:
How about naming it "Essential Test", "Basic Test", or "Core Test"? These names seem more in line with "Demo Test" and "Rune Test". Also, since `make test` is not the only task that this Github action does, "Make Test" is not a name that best summarizes this action.

Reply:
Renamed to "Essential test".